crates/epaint_default_fonts/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+//! A library containing built-in fonts for `epaint`, embedded as bytes.
+//!
+//! This is intended to be consumed through the `epaint` crate.
+
+pub const HACK_REGULAR: &[u8] = include_bytes!("../fonts/Hack-Regular.ttf");
+pub const NOTO_EMOJI_REGULAR: &[u8] = include_bytes!("../fonts/NotoEmoji-Regular.ttf");
+pub const UBUNTU_LIGHT: &[u8] = include_bytes!("../fonts/Ubuntu-Light.ttf");
+pub const EMOJI_ICON: &[u8] = include_bytes!("../fonts/emoji-icon-font.ttf");
```

Comment:
For bonus points, add some nice docstrings to these :) But only if you want

Reply:
I'll see what I can come up with without having much context on these choices :)